### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by ethanjurman**
The main page is now using material ui
image

if you click see calendar details you get to this page: (note css stylings are a tad off)
image

each component should now export a peek view and a page view (if it has a page). we'll use the peek view for the main home page and page view when we click "see details"

### CODE REVIEW

**Comment by evajurman** [comment]
you must run `npm install` for many of the libraries to work

File: src/client/css/styles.css [hunk 1]
```diff
@@ -2,3 +2,40 @@
...
+}
+
+.rbc-btn-group {
+  text-align: center;
+}
+
+.rbc-toolbar-label {
+  text-align: center;
+  flex: 1;
+}*/
```
Comment (CShapleigh) [inline src/client/css/styles.css]: why is all of this commented out?

Comment (evajurman) [inline src/client/css/styles.css]: oh my bad I was testing out calendar ui stuff... It should be removed

**Comment by CShapleigh** [comment]
Resolve conflicts, then I'd go ahead with a merge.

**Comment by CShapleigh** [comment]
:100: / :100:

**Pull request merged**